vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -357,4 +359,55 @@ public void addThemeVariants(ComboBoxVariant... variants) {
     public void removeThemeVariants(ComboBoxVariant... variants) {
         HasThemeVariant.super.removeThemeVariants(variants);
     }
+
+    /**
+     * Adds the given components as children of this component at the slot
+     * 'prefix'.
+     *
+     * @param components
+     *            The components to add.
+     * @see <a href=
+     *      "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot">MDN
+     *      page about slots</a>
+     * @see <a href=
+     *      "https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element">Spec
+     *      website about slots</a>
+     */
+    protected void addToPrefix(Component... components) {
```

Comment:
Should those be re-added with `@Deprecated`? I don't think you wanna maintain those APIs the future or?

Reply:
I'm not sure, we might consider that later together with other components that have this method.